app/src/main/java/com/cmput301f18t20/medicalphotorecord/Problem.java

```diff
@@ -3,15 +3,15 @@
 import android.location.Location;
 
 import java.util.ArrayList;
+import java.util.Date;
 
 public class Problem extends Entry implements Refreshable {
     protected String description;
     protected ArrayList<Record> records = new ArrayList<>();
 
```

Comment:
Can we keep the 0-argument constructor available for testing?

Reply:
Yup